ontology/yaml/resources/METERS/entity_types/EM.yaml

```diff
@@ -20,6 +20,7 @@ EM_PWM:
   - EM
   - PWM
 
+
```

Comment:
Please revert if this is just whitespace.

Reply:
This is just a white space...

Reply:
white space deleted